docs/src/examples/components/Carousel/Types/CarouselExample.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react'
+import { Carousel, Image } from '@stardust-ui/react'
+
+const carouselItems = [
+  { key: 'ade', slide: { as: Image, src: 'public/images/avatar/large/ade.jpg' } },
```

Comment:
would
```
slide: <Image src={} />
```
work as well?
do we need to pass something down to the item from the carousel?